### PR TITLE
Add proper Bower support

### PR DESCRIPTION
**Pull request by ahmedalsudani**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -7,18 +7,14 @@
...
-  "dependencies": {"ramda": "latest"},
-  "main": [
-    "lib/src/hack-stack/hack-stack.js",
-    "lib/src/hack-stack/hack-stack-mock-service.js",
-    "lib/src/hack-stack/hack-stack-wrapper-service.js",
-    "lib/src/hack-stack/hack-stack-mockfh-service.js",
-    "lib/src/hack-stack/hack-stack-utils-service.js"
-  ],
+  "dependencies": {
+    "ramda": "latest"
```
Comment (brianolynyk) [inline bower.json]: I would lock down the ramda version.  They've been pretty bad about changing their API between versions which could break the hackstack unexpectedly.

Comment (ahmedalsudani) [inline bower.json]: Oh yes, good catch

File: bower.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-  "dependencies": {"ramda": "latest"},
-  "main": [
-    "lib/src/hack-stack/hack-stack.js",
-    "lib/src/hack-stack/hack-stack-mock-service.js",
-    "lib/src/hack-stack/hack-stack-wrapper-service.js",
-    "lib/src/hack-stack/hack-stack-mockfh-service.js",
-    "lib/src/hack-stack/hack-stack-utils-service.js"
-  ],
+  "dependencies": {
+    "ramda": "~0.14.0"
```
Comment (brianolynyk) [inline bower.json]: I meant to LOCK this down.  No ~.

Comment (ahmedalsudani) [inline bower.json]: Fixed

File: bower.json [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
-  "dependencies": {"ramda": "latest"},
-  "main": [
-    "lib/src/hack-stack/hack-stack.js",
-    "lib/src/hack-stack/hack-stack-mock-service.js",
-    "lib/src/hack-stack/hack-stack-wrapper-service.js",
-    "lib/src/hack-stack/hack-stack-mockfh-service.js",
-    "lib/src/hack-stack/hack-stack-utils-service.js"
-  ],
+  "dependencies": {
+    "ramda": "0.14.0"
```
Comment (brianolynyk) [inline bower.json:15]: :+1:

**Comment by brianolynyk** [comment]
Just waiting on an answer to the license question.

**Comment by brianolynyk** [comment]
Looks good.  Squash and I'll merge.

**Pull request merged**